python/lsst/ip/isr/isrTask.py

```diff
@@ -1611,6 +1665,31 @@ def ensureExposure(self, inputExp, camera=None, detectorNum=None):
 
         return inputExp
 
+    @staticmethod
+    def extractCalibDate(calib):
+        """Extract common calibration metadata values that will be written to
+        output header.
+
+        Parameters
+        ----------
+        calib : `lsst.afw.image.Exposure` or `lsst.ip.isr.IsrCalib`
+            Calibration to pull date information from.
+
+        Returns
+        -------
+        dateString : `str`
+            Calibration creation date string to add to header.
+        """
+        if hasattr(calib, "getMetadata"):
+            if 'CALIB_CREATION_DATE' in calib.getMetadata():
+                return " ".join((calib.getMetadata().get("CALIB_CREATION_DATE", "Unknown"),
```

Comment:
Presumably the `CALIB_CREATION_DATE` vs `CALIB_CREATE_DATE` is to do with versioning (I see you changing it on the other ticket), so maybe just a comment here about why this tried this first?